erpnext/www/support/index.py

```diff
@@ -0,0 +1,78 @@
+from __future__ import unicode_literals
+import frappe, json
+from frappe import _
+
+def get_context(context):
+	context.no_cache = 1
+	context.align_greeting = ''
+	setting = frappe.get_doc("Support Settings")
+
+	context.greeting_title = setting.greeting_title
+	context.greeting_subtitle = setting.greeting_subtitle
+	
+	# Support content
+	context.favorite_article_list=[]
+	context.help_article_list=[]
+	context.category_list = frappe.get_all("Help Category", fields="name")
+	favorite_articles = get_favorite_articles_by_page_view()
+	if not favorite_articles or len(favorite_articles) < 3:
```

Comment:
```suggestion
	if not favorite_articles:
```

If you get a list, show it anyway, or add latest articles next to it. So if you get 2 favorite articles, add 4 recent ones